lib/pengine/native.c

```diff
@@ -545,16 +549,13 @@ native_print(resource_t * rsc, const char *pre_text, long options, void *print_d
 
         /* Ignore target role Started, as it is the default anyways
          * (and would also allow a Master to be Master).
-         * Show if current role differs from target role,
-         * or if target role limits our abilities. */
+         * Show if target role limits our abilities. */
         if (target_role_e == RSC_ROLE_STOPPED) {
             flagOffset += snprintf(flagBuffer + flagOffset, LINE_MAX - flagOffset, "%sdisabled", flagOffset?", ":"");
             rsc->cluster->disabled_resources++;
 
         } else if (uber_parent(rsc)->variant == pe_master
-                   && target_role_e > RSC_ROLE_STOPPED
-                   && target_role_e < RSC_ROLE_MASTER
-                   && safe_str_neq(target_role, role2text(rsc->role))) {
+                   && target_role_e == RSC_ROLE_SLAVE) {
```

Comment:
FYI, I just noticed the documentation says that Started does constrain m/s resources: "Allow the resource to be started (In the case of multi-state resources, they will not be promoted to master)". However, testing shows that is incorrect in practice, so I'll update the documentation. I'm not sure whether that behavior changed at some point, and if so, whether that was intentional ...

Fun trivia: any invalid values are logged and treated as the default 'Started' (e.g. target-role=foo). Mis-capitalized values are not logged, but are treated as default ... so target-role=stopped will silently be treated as Started. I'm thinking the value should be case-insensitive, but not worrying about it right now.